fuzz/install

```diff
@@ -38,3 +38,10 @@ make install
 cd ..
 rm -rf afl-${AFL_VERSION}
 rm afl-${AFL_VERSION}.tgz
+
+# build stellar-core under fuzzer
+[[ -f stellar-core/autogen.sh ]] || git clone https://github.com/stellar/stellar-core stellar-core
```

Comment:
This check doesn't really need to be here if we're doing it at image build time. The idea here was we could mount some source from the host but I don't think we really need that. Plus at build time it's not even an option.